modules/apps/portal-odata/portal-odata-impl/src/test/java/com/liferay/portal/odata/internal/filter/ExpressionConvertImplTest.java

```diff
@@ -281,16 +279,20 @@ public void testConvertMemberExpressionWithLambdaAnyEqOnCollectionField()
 
 		@Override
 		public Map<String, EntityField> getEntityFieldsMap() {
-			return Stream.of(
-				new CollectionEntityField(
-					new StringEntityField(
-						"keywords", locale -> "keywords.raw")),
-				new StringEntityField("title", locale -> "title"),
-				new DateTimeEntityField(
-					"dateTime", locale -> "dateTime", locale -> "dateTime")
-			).collect(
-				Collectors.toMap(EntityField::getName, Function.identity())
-			);
+			EntityField collectionEntityField = new CollectionEntityField(
+				new StringEntityField("keywords", locale -> "keywords.raw"));
+			EntityField stringEntityField = new StringEntityField(
+				"title", locale -> "title");
+			EntityField dateTimeEntityField = new DateTimeEntityField(
+				"dateTime", locale -> "dateTime", locale -> "dateTime");
+
+			return HashMapBuilder.put(
+				collectionEntityField.getName(), collectionEntityField
+			).put(
+				stringEntityField.getName(), stringEntityField
+			).put(
+				dateTimeEntityField.getName(), dateTimeEntityField
+			).build();
 		}
 
```

Comment:
@W2jx, for this commit and the other similar commits, please inline these. We can know `collectionEntityField.getName()` value.